### PR TITLE
DOCS-9998-sftp-index-studio-editorial-k

### PR DESCRIPTION
**Pull request by kevintroller**
SFTP Connector Index and Studio configuration documentation for editorial

### CODE REVIEW

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+= SFTP Connector 1.3 - Mule 4
+:page-aliases: connectors::sftp/sftp-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for SFTP (SFTP Connector) provides access to files and folders on a SFTP server. SFTP connector operations manage file transfers over the SFTP (Secure File Transfer Protocol) protocol. The connector main features include:
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]: "a SFTP server" > "an SFTP server," if people pronounce this ess-eff-tee-pee

Comment (kbostrom) [reply sftp/1.3/modules/ROOT/edits/index.adoc]: connector main features > connector's main features

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+* Read files or list directory contents on demand.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]:
```suggestion
* The ability to read files or list directory contents on demand
```

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* Support for common SFTP operations such as creating directories, copying, moving, renaming, deleting files.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]:
```suggestion
* Support for common SFTP operations such as creating directories, copying, moving, renaming, and deleting files
```

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* Support for locking files.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]:
```suggestion
* Support for locking files
```

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* File matching functionality.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]:
```suggestion
* File matching functionality
```

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+* Consistent design with the File and FTP Connectors. The same set of operations is available on these connectors, and they behave similarly.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]:
```suggestion
* A design that is consistent with that of File Connector and FTP Connector
+
The same set of operations is available on these connectors, and they behave similarly to SFTP Connector.
```

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+=== Representation of File Attributes
+
+Mule 4 provides a message structure composed of a payload and attributes, where the data moves around. This means that the payload is a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, timestamps, and so on. For the FTP and SFTP connectors these attributes are known as the _file attributes_.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]:
```suggestion
Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are known as  _file attributes_.
```
GLOBALLY omit ", and so on." It is vague and also unnecessary if the text includes "such as," which implies some but not all.

File: sftp/1.3/modules/ROOT/edits/index.adoc [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+The Mule message contains the concepts of encoding and MIME type. These describe the format in which a payload is represented. The file attributes describe a file’s encoding and MIME type.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/index.adoc]: Does a "message" contain "concepts" or are we saying that the message uses encoding and MIME type? Clarify.

Comment (kbostrom) [reply sftp/1.3/modules/ROOT/edits/index.adoc]: These three sentences together include some repetition: can you pare it down?

**Review by kbostrom — APPROVED** [review]
CE first file in PR done

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with SFTP Connector:
+
+* On New or Updated File +
+Initiates a flow by polling a directory for files that have been created or updated. A message is generated for each file that is found.
+* HTTP Listener +
+Initiates a flow each time it receives a request on the configured host and port
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
Initiates a flow each time it receives a request on the configured host and port.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+* Scheduler +
+Initiates a flow when a time-based condition is met
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
Initiates a flow when a time-based condition is met.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with SFTP Connector:
+
+* On New or Updated File +
+Initiates a flow by polling a directory for files that have been created or updated. A message is generated for each file that is found.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]: attempt to make one sentence, active voice
```suggestion
Initiates a flow by polling a directory for files that have been created or updated and generates a message for each file found.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+
+For example, to configure the On New or Updated File source, follow these steps:
+
+. In the *Mule Palette* view, select *On New or Updated File*.
+. Drag *On New or Updated File* to the Studio canvas.
+. On the *On New or Updated File* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of *On New or Updated File* in the app.
+. On the *General* tab, specify the connection information for the connector.
+. Set the *Host* field to specify the SFTP server host, for example `localhost`, `192.168.0.1`, and so on.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
. Set the *Host* field to specify the SFTP server host, for example, `localhost` or `192.168.0.1`.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. On the *On New or Updated File* configuration screen, specify a *Scheduling Strategy*, and other optional fields.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
. On the *On New or Updated File* configuration screen, specify a *Scheduling Strategy* value and optionally complete other fields.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to configure the SFTP server connection.
+
+To configure the global element for SFTP Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, specify the connection information for the connector:
+. Set the *Working Directory* as the root of every relative path used with this connector. If it's not provided, the value defaults to the remote server.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
. Set the *Working Directory* as the root of every relative path used with this connector. 
+
If it's not provided, the value defaults to the remote server.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+. Set the *Host* field to specify the SFTP server host, for example `localhost` or `192.168.0.1`.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
. Set the *Host* field to specify the SFTP server host, for example, `localhost` or `192.168.0.1`.
```

File: sftp/1.3/modules/ROOT/edits/sftp-studio.adoc [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+. On the *Advanced* tab, optionally specify timeout configuration and reconnection strategy.
```
Comment (kbostrom) [inline sftp/1.3/modules/ROOT/edits/sftp-studio.adoc]:
```suggestion
. On the *Advanced* tab, optionally specify a timeout configuration and reconnection strategy.
```

**Review by kbostrom — DISMISSED** [review]
CE done

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**